packages/@dcl/sdk/src/cli/index.ts

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env node
+
+import { readdirSync } from 'fs'
+import { resolve } from 'path'
+
+import { CliError } from './error'
+
+// leaving args as "any" since we don't know yet if we will use them
+type FileFn = (...args: any) => Promise<void>
+
+interface FileFns {
+  help?: FileFn
+  default?: FileFn
+}
+
+const JS_EXT = '.js'
+const COMMANDS_PATH = resolve(__dirname, './commands')
+
+const isJSFile = (file: string): boolean => file.slice(-3) === JS_EXT
+
+const commands = new Set(
+  readdirSync(COMMANDS_PATH)
+    .filter((fileName) => isJSFile(fileName))
+    .map((fileName) => fileName.slice(0, -3))
+)
+
+const listCommandsStr = () =>
+  Array.from(commands)
+    .map(
+      ($) => `
+  * npx sdk ${$}`
```

Comment:
Should it be @dcl/sdk instead? https://www.npmjs.com/package/sdk

```suggestion
  * npx @dcl/sdk ${$}`
```